pkg/provider/ip/provider.go

```diff
@@ -75,6 +78,11 @@ func (p *ipv4Provider) InitResource(instance ec2.EC2Instance) error {
 	eniManager := eni.NewENIManager(instance)
 	presentIPs, err := eniManager.InitResources(p.apiWrapper.EC2API)
 	if err != nil {
+		if errors.Is(err, utils.ErrNotFound) {
+			msg := fmt.Sprintf("The instance type %s is not supported for Windows yet", instance.Type())
```

Comment:
nit: I think you can drop the "yet" in the warning. That implies it will be supported some day, which it may not

Reply:
Makes sense.